docs/fans.md

```diff
@@ -0,0 +1,36 @@
+
+## ** Why does SHE do it? **![trophy](https://lh3.googleusercontent.com/GHhpehfDWMyYfynU_GNYPlwimDWV3QazaqU9raCCvK0wstXGhMvy5HXBNbWRrNQUyZoAsXrA_FAxSrPFq0mMYAl76Nco4WtJNtNWP11jDWcZRZYmqsoZhhIHorlkr6SfrCRhVPIH)
+
+** Fan ** = "A person who has a strong interest in or admiration for a particular person or thing.”
+
+Now that you know the definition of a fan, internalize it, and ask yourself, what would I do to meet my favorite athlete? How incredible it would be to have specialized content from them? Image the feeling of having your favorite athlete call out your name right before landing a perfect dive. Then having that video moment attached to a digital collectible card forever!  Shenanigan encourages supporting your athlete through donations! It gives athletes the chance to pursue their dreams and fans the chance to earn, while helping and interacting on a more personal level! Shenanigan provides each fan with the web3 instruments necessary to achieve this. Now get out there and grow with your athletes! 
+
+## ** How do you do it? ** ![iphone](https://lh6.googleusercontent.com/PUZX7_Li0FoaCsFyqfU1qve0nbcAQO1lBtNWdltyEyWKaB_RhCyYeJsYIBBJguKMf1qo9kYNdx2c1eLZoBlCojaNMeO1ShWsG93PbiFHf2wC5FuBsvKvTqdVWzs0QLmlwO09Q1ik)
+
+Our fans download the "Shenanigan Mobile App" and register for a fan account. Once registered as a fan the user will interact with the easy to use app interface. User's search for their favorite athlete and once found they can join the athlete's fan club. Once joined users can interact with the community and their favorite athletes! The more a user interacts with their athletes of choice and corresponding fan club's the higher the chance of earning rewards and finding rare digital collectibles known as an “NRG Challenge cards.”
+
+*For more indepth of NRG Collectable cards read more at https://shenanigan.substack.com/p/she-teaches-you-about-dathletes*
+
+## * How do they benefit? *![moneybag](https://lh6.googleusercontent.com/HnldgSWanuvnisv_IkMKUvnoEWkk5KIrPl5C52bF0Ft_5qalW7koarsiXUD9DnFV_miiufmxBomIzOgy8SCAPMoobAqtSYhpHIGcapJYvH6ZSysfGn5o224WhixrD9DelON13mPO)
+
+SHE generates a community where fans can directly donate to the success of their favorite athletes while watching their progress and collecting rare collectibles along the way! By donating to one's favorite athletes, fans will be rewarded with first call to limited digital collector cards! Fans can earn "Fan Tokens", by simply holding their favorite athletes trading card.
+
+Fan Tokens are a native in app token which can be used to tip their favorite athletes or level up their digital collector cards increasing their value. Fan tokens can also be locked up (staked) to earn APY! 
+
+SHE also creates a market of challenges and predictions to gamify live broadcasts where spectators (fans) can be incentivized in various ways:
```

Comment:
NIT: I think this is good, but it's not quite a sales pitch. Keep it for now but we should look into explaining this

Reply:
I changed the verbiage on this so let me know what you think of the new presentation.